types/http.d.ts

```diff
@@ -108,6 +108,11 @@ export interface HttpRequest {
      */
     bufferBody?: Buffer;
 
+    /**
+     * Get the value of a particular header field
+     */
+    get(field: string): string | undefined;
```

Comment:
So this property was already there just not explicitly there in the types?

Reply:
Yes. In general, the types are pretty lacking